scripts/images/rancher-images.txt

```diff
@@ -1,25 +1,35 @@
 docker.io/rancher/fleet-agent:v0.13.0
 docker.io/rancher/fleet:v0.13.0
+docker.io/rancher/kubectl:v1.29.2
```

Comment:
We need this since the current version of rancher-logging & rancher-monitoring relies on `rancher/kubectl:v1.29.2`.